apps/demos/Demos/Menu/Scrolling/Vue/data.ts

```diff
@@ -1,338 +1,338 @@
 const products = [
-  {
+    {
```

Comment:
Why did you change formatting here? In ContextMenu data uses indent with 2 spaces

Reply:
oh i didnt noticed this ill fix this one

Reply:
[42b9220](https://github.com/DevExpress/DevExtreme/pull/27440/commits/42b9220d05d00d98695c0aad1ddd96f48bb18e51)